client/tailscale/tailscale.go

```diff
@@ -5,8 +5,10 @@
 
 // Package tailscale contains a Go client for the Tailscale control plane API.
 //
-// Warning: this package is in development and makes no API compatibility
-// promises as of 2022-04-29. It is subject to change at any time.
+// This package is only intended for internal and transitional use.
```

Comment:
> transitional use

So I guess it should still be fine to add new functionality here for which the API is not yet sufficiently stable to add it in the official client, but that something in OSS already needs to consume with the goal of eventually moving into the official client.
(I'm referring to VIPServices API for which we added a shim in the operator https://github.com/tailscale/tailscale/blob/main/cmd/k8s-operator/tsclient.go. I don't necessarily think that such use cases will be frequent)

Reply:
I think the best place to put this is actually in `internal/client/tailscale`. I'll do that in this PR to show the way.

Reply:
@irbekrm I ended up doing this in a [separate PR](https://github.com/tailscale/tailscale/pull/15001).